packages/frontend/src/components/HourInput.vue

```diff
@@ -6,33 +6,49 @@
       :time-entrie="timeEntrie"
       @click="onTimeLeftInDayClick"
     />
-    <input
-      ref="inputRef"
-      v-model="value"
-      :class="{ error, nonZero, hovered }"
-      type="text"
-      novalidate
-      inputmode="decimal"
-      :disabled="!isOnline || isLocked"
-      @mouseover="hovered = !(!isOnline || isLocked)"
-      @mouseleave="hovered = false"
-      @input="onInput"
-      @touchstart="onTouchStart"
-      @blur="onBlur"
-      @focus="onFocus"
-      @click="onClick"
-    />
+    <div class="input-wrapper">
+      <input
+        ref="inputRef"
+        v-model="value"
+        :class="{ 'has-comment': hasComment, error, nonZero, hovered }"
+        type="text"
+        novalidate
+        inputmode="decimal"
+        :disabled="!isOnline || isLocked"
+        @mouseover="hovered = !(!isOnline || isLocked)"
+        @mouseleave="hovered = false"
+        @input="onInput"
+        @touchstart="onTouchStart"
+        @blur="onBlur"
+        @focus="onFocus"
+        @click="onClick"
+        v-click-outside="handleOutsideClick"
+      />
+      <span v-if="hasComment" class="comment-icon">💬</span>
+
+      <HourCommentBox
```

Comment:
This is a good solution. Here we render one comment box per input field. However, if we were making a super high performance frontend, we would probably just have one comment-box for the whole view and then just position the box using javascript. This would render less divs making the browser happier and it would require less rendering from Vue.

That would make the code a whole lot more complex. This is not relevant for this PR, but is good to know when you develop for a customer :+1: